### PR TITLE
Decode HTML entities and shorten muppet removal

### LINKED ISSUE #113: in #denied-dupes it shows &amp; instead of just '&'
As the title says, In #denied-dupes it will show &amp: instead of just '&'

Idk if it's something fixable or not.

image

### PR DESCRIPTION
**Pull request by ThaTiemsz**
Fixes #113

### CODE REVIEW

File: Commands/Commands/queue.js [hunk 1]
```diff
@@ -374,7 +376,7 @@ commands.registerVote = {
                         name: 'f1'
                       }, user.id))
                     })
-                  }, 2500))
+                  }, 500))
```
Comment (Arzte) [inline Commands/Commands/queue.js:379]: why?

Comment (ThaTiemsz) [reply Commands/Commands/queue.js:379]: I often still see some f1 reactions on suggestions and it's pissing me off

Comment (Arzte) [reply Commands/Commands/queue.js:379]: They still get removed though, its a 2.5 delay... its not that long.

Comment (ThaTiemsz) [reply Commands/Commands/queue.js:379]: Sometimes they don't, I manually removed around 5 of them like an hour or so ago

**Review by Dougley — APPROVED** [review]

**Pull request merged**